src/Microsoft.Tye.Core/KubernetesManifestGenerator.cs

```diff
@@ -26,6 +26,9 @@ internal static class KubernetesManifestGenerator
             var metadata = new YamlMappingNode();
             root.Add("metadata", metadata);
             metadata.Add("name", project.Name);
+            if (!String.IsNullOrEmpty(project.Namespace)) {
```

Comment:
```suggestion
            if (!string.IsNullOrEmpty(project.Namespace))
            {
```

Reply:
@brendandburns - if you run `dotnet format` at the command line it should autoformat stuff for you. You may need to run `dotnet tool restore` to bring down the tool. I'm not sure if the build script installs it or not.

Reply:
done.